terraform/account/credentials.tf

```diff
@@ -12,55 +12,14 @@ terraform {
 variable "default_role" {
   default = "opg-refunds-ci"
 }
-variable "management_role" {
-  default = "opg-refunds-ci"
-}
-variable "old_account_default_role" {
-  default = "opg-refunds-ci"
-}
 
 
 provider "aws" {
-  version = "2.70.0"
-  region  = "eu-west-1"
+  region = "eu-west-1"
```

Comment:
Version no longer required?

Reply:
yep - there's a different way of doing this in the future.